orchestration/api/instances.py

```diff
@@ -164,6 +164,14 @@ def wf_ops(tenant_id='', instance_id=''):
         if instance_id == '':
```

Comment:
it's better to divide this function to different function, so that you don't have to write if-esle statements.